app/views/embed/body/_was_seed.html.erb

```diff
@@ -1,21 +1,35 @@
 <div class='sul-embed-body sul-embed-was-seed-container'
       data-sul-embed-theme="<%= viewer.asset_url('was_seed.css') %>"
       data-sul-icons="<%= viewer.asset_url('sul_icons.css') %>">
-  <div class='sul-embed-was-seed' data-sul-thumbs-list-count="<%= viewer.thumbs_list.length %>">
-    <ul class='sul-embed-was-thumb-list'>
-      <% viewer.thumbs_list.each do |thumb_record| %>
-        <li class='sul-embed-was-thumb-item'>
-          <div class='sul-embed-was-thumb-item-div'>
-            <img class='sul-embed-was-thumb-item-img' src="<%= thumb_record['thumbnail_uri'] %>">
-            <div class='sul-embed-was-thumb-item-date'>
-              <a href="<%= thumb_record['memento_uri'] %>" target='_parent'>
-                <%= viewer.format_memento_datetime(thumb_record['memento_datetime']) %>
-              </a>
+  <div class='sul-embed-was-seed'>
+    <div class='sul-embed-was-seed-content'>
+      <div class='sul-embed-was-seed-row'>
+        <div class='sul-embed-was-seed-column sul-embed-was-seed-info'>
+          <%= link_to viewer.archived_site_url, target: '_top' do %>
+            <div class="sul-embed-was-seed-preview">
+              <%= image_tag viewer.shelved_thumb, alt: '' %>
             </div>
-          </div>
-        </li>
-      <% end %>
-    </ul>
+            Stanford Web Archive Portal
+          <% end %>
+        </div>
+        <div class='sul-embed-was-seed-column sul-embed-was-seed-captures'>
+          <% if viewer.capture_list.length.positive? %>
+            <h1>
+              <%= "Captured #{pluralize(viewer.capture_list.length, 'time')} between #{viewer.format_memento_datetime(viewer.capture_list.first.datetime)} and #{viewer.format_memento_datetime(viewer.capture_list.last.datetime)}" %>
+            </h1>
+          <% end %>
+          <ul class='sul-embed-was-seed-list'>
+            <% viewer.capture_list.each do |memento| %>
+              <li class='sul-embed-was-seed-list-item'>
+                <%= link_to memento.url, target: '_top', class: 'sul-embed-stretched-link' do %>
+                  <%= DateTime.parse(memento.datetime).utc %>
```

Comment:
There is a test (and your gif in the PR) that expect a GMT formatted date time but it looks like we're explicitly using UTC here. 

<img width="948" alt="Screen Shot 2020-07-31 at 4 25 39 PM" src="https://user-images.githubusercontent.com/96776/89087874-a0616700-d34a-11ea-9bb0-590b346f1c22.png">

What's the expected formatting for the momento crawl times? (@jvine / @mejackreed)

Reply:
Per stakeholder feedback, we were asked to use UTC.

Reply:
Perfect, thanks @anarchivist.